tests/components/zwave_js/conftest.py

```diff
@@ -1002,39 +1022,39 @@ def climate_radio_thermostat_ct100_mode_and_setpoint_on_different_endpoints_fixt
 
 
 @pytest.fixture(name="vision_security_zl7432")
-def vision_security_zl7432_fixture(client, vision_security_zl7432_state):
+def vision_security_zl7432_fixture(client, vision_security_zl7432_state) -> Node:
```

Comment:
If you feel fancy, we could also look into typing the parameters

Reply:
Not today :)